pkg/utils/exec/exec.py

```diff
@@ -0,0 +1,52 @@
+
+from kubernetes.stream import stream
+
+# PodDetails contains all the required variables to exec inside a container
+class PodDetails(object):
+	def __init__(self, PodName=None, Namespace=None, ContainerName=None):
+		self.PodName      = PodName
+		self.Namespace       = Namespace
+		self.ContainerName = ContainerName
+
+# checkPodStatus verify the status of given pod & container
+def checkPodStatus(pod , containerName):
+
+    if pod.status.phase.lower() != "running":
+        return ValueError("%s pod is not in running state, phase: %s", pod.Name, pod.Status.Phase)
+
+    for container in pod.status.container_statuses:
+        if container.name == containerName and container.state.terminated != None:
+            return ValueError("%s container of %s pod is not in ready state, phase: %s", container.name, pod.metadata.name, pod.status.phase)
+
+    return None
+
+def Exec(commandDetails, clients , command):
+    name=commandDetails.PodName
```

Comment:
Do we need an extra variable here, can't we directly use commandDetails.PodName? -- as we are not modifying its value in this function